config/routes.rb

```diff
@@ -8,6 +8,12 @@
   get "/authorized", to: "sessions#show"
   post "/reservation", to: "reservations#create"
 
+  #User Routes
+  #my reservations Routes
+  get "/user/reservations", to: "reservations#my_reservations"
+  #delete reservation
+  delete "/user/reservations", to: "reservations#delete_reservation"
```

Comment:
[OPTIONAL] please add the correct indentation to follow the admin routes and make the routes readable and clean

Reply:
@ichala Done